lib/asciidoctor/js/path_resolver.rb

```diff
@@ -0,0 +1,15 @@
+# encoding: UTF-8
+module Asciidoctor
+class PathResolver
+  SLASH = '/'
+  BACKSLASH = '\\'
+  WindowsRootRx = /^[a-zA-Z]:(?:\\|\/)/
+  WebRootRx = /^(file|https?):\/\//
+
+  def is_root? path
+    (path.start_with? SLASH) ||
+      (::JAVASCRIPT_IO_MODULE == 'xmlhttprequest' && (WebRootRx.match? path)) ||
```

Comment:
Remove when Asciidoctor 1.5.6.2 will be released

Reply:
Really I think we should wait on this PR until 1.5.6.2 is available, which could be this week.

Reply:
Ok